ui/src/overridden/backoffice/Importer/Importer.js

```diff
@@ -46,15 +214,10 @@ export default class Importer extends Component {
     try {
       const response = await importerApi.createTask(formData);
 
-      this.setState({
-        provider: '',
-        mode: '',
-        file: null,
-        postDone: true,
-        taskId: response.data.id,
-        error: {},
-      });
+      const taskId = response.data.id;
+      history.push(BackOfficeRouteGenerators.importerDetailsFor(taskId));
```

Comment:
```suggestion
      goTo(BackOfficeRouteGenerators.importerDetailsFor(taskId));
```

Reply:
Hm for this one the reason I couldn't use `goTo` is because it was not exported!